xline/src/storage/lease_store/mod.rs

```diff
@@ -347,14 +180,15 @@ where
 {
     /// New `LeaseStoreBackend`
     pub(crate) fn new(
```

Comment:
Same as above, please remove the `pub(crate)`

Reply:
Removed